Show other websites for meetups

plaindocs opened the pull request:
@rosewms this replaces the single `website` field with a `links` dict, allowing multiple links with custom labels:

```yaml
links:
  Meetup: https://www.meetup.com/write-the-docs-pdx/events/
  LinkedIn: https://linkedin.com/groups/...
```

Each entry is rendered as a list item with the key as the visible link text.

Previously:

- ~~can only have one website~~
- ~~cannot customize the link title~~

Now:

- can have multiple links
- can customize each link's label text
- still need a meetup ID


----
📚 Documentation preview 📚: https://writethedocs-www--2242.org.readthedocs.build/

plaindocs committed:
We already have this data in the meetup key

plaindocs committed:
Merge branch 'main' into sam-meetups

ericholscher commented on docs/_templates/include/meetups/listing.jinja:
```diff
@@ -35,6 +35,13 @@
             {% endif %}
 
+            {# Other internet presence  #}
+            {% if meetup['website'] %}
```
I wonder about doing this with `links` and allowing multiple?

Something like:

```
- links:
  - Meetup: https://meetup...
  - LinkedIn: https://linked..
```

That way you can control the link copy and allow multiple if needed?

ericholscher commented on docs/_templates/include/meetups/listing.jinja:
```diff
@@ -35,6 +35,13 @@
+              <div>
+                Elsewhere: <a href="{{ meetup['website']}}">{{ meetup['website'] }}</a>
+              </div>
+            {% endif %}
```
```suggestion
{% if meetup['links'} %}
<ul>
{% for text, link in meetup['links'].items() %}
<li>
 <a href="{{ link }]">{{ text }}</a>
</li>
{% endfor %}
</ul>
{% endif %}
```

rosewms commented:
@ericholscher and @plaindocs could we potentially change how we frame it away from Meetups and list them as Community links and allow multiple links?

plaindocs commented:
Yep, I've got two half assed fixes to this, I'll clean one up so we can put it in and forget about it.

rosewms commented:
@plaindocs we never merged a solution and the organizer just reached out! Did you implement the workarounds or do we need to start over?

plaindocs commented:
Let me fight all the current fires and see if I can kick this into shape after.

ericholscher commented:
@copilot address review feedback here and fix it up.

Copilot committed:
Replace single website field with links dict for meetup listings

Agent-Logs-Url: https://github.com/writethedocs/www/sessions/b77b9037-9868-40c4-92f3-586c13c09ca7

Co-authored-by: ericholscher <25510+ericholscher@users.noreply.github.com>

ericholscher commented:
Opened a new PR without conflicts. 

https://github.com/writethedocs/www/pull/2607
